apisix/plugins/response-rewrite.lua

```diff
@@ -126,6 +177,25 @@ function _M.body_filter(conf, ctx)
         return
     end
 
+    if conf.filters then
+
+        local body = core.response.hold_body_chunk(ctx)
+        if not body then
+            return
+        end
+
+        for _, filter in ipairs(conf.filters) do
+            if filter.scope == "once" then
+                body = re_sub(body, filter.regex, filter.replace, filter.options)
```

Comment:
should we check the result of `re_sub`? like: https://github.com/apache/apisix/blob/51eec3ea64bba464ab4a6741acc318c502cab26c/apisix/plugins/proxy-rewrite.lua#L101-L106

Reply:
This result `err` places on `check_schema`, had checked it.

If check on `body_filter` phase, `err` exception found, should update response status code like `500`, but it's too late here.

Reply:
Enn, but we can log it, right?

Reply:
yes, at least there is a log.

Reply:
Done.